### PR TITLE
solvedDay 14 q2: Search in a BST #348

### PR DESCRIPTION
**Pull request by Kalpana8055**
Screenshot (6)

### CODE REVIEW

File: Day-14/q2-Search in a BST/solution.c++
Comment (bh-g) [inline Day-14/q2-Search in a BST/solution.c++:1]: Hey Kalpana, pls rename this file to Kalpana8055--C.md, (username--language.md)

Comment (Kalpana8055) [reply Day-14/q2-Search in a BST/solution.c++:1]: ok

Comment (Kalpana8055) [reply Day-14/q2-Search in a BST/solution.c++:1]: @bh-g please merge this PR

Comment (Kalpana8055) [reply Day-14/q2-Search in a BST/solution.c++:1]: @Nitya-Pasrija please merge this pr

**Pull request merged**